AdvLoggerPkg/UnitTests/AdvancedLoggerWrapper/AdvancedLoggerWrapperTestApp.c

```diff
@@ -392,47 +389,31 @@ TestCursorWrappingMP (
       );
 
     // Now check the index
-    EndPointer = &mMessageEntry.Message[ADV_TIME_STAMP_PREFIX_LEN + sizeof (ADV_WRAP_TEST_STR) + 8 - 1];
-    Status     = AsciiStrHexToUintnS (
-                   &mMessageEntry.Message[ADV_TIME_STAMP_PREFIX_LEN + sizeof (ADV_WRAP_TEST_STR) - 1],
-                   &EndPointer,
-                   &StrIndex
-                   );
-    if (EFI_ERROR (Status)) {
-      UtStatus = UNIT_TEST_ERROR_TEST_FAILED;
-      goto Done;
-    }
-
-    if (StrIndex >= NumberOfProcessors) {
-      // Index is out of range
-      UtStatus = UNIT_TEST_ERROR_TEST_FAILED;
-      goto Done;
-    }
-
-    if (TempCache[StrIndex] == 0) {
-      // Printed this index already
-      UtStatus = UNIT_TEST_ERROR_TEST_FAILED;
-      goto Done;
-    } else {
-      TempCache[StrIndex] = 0;
-    }
+    EndChar                                                           = mMessageEntry.Message[ADV_TIME_STAMP_PREFIX_LEN + PrefixSize + 8];
+    mMessageEntry.Message[ADV_TIME_STAMP_PREFIX_LEN + PrefixSize + 8] = '\0';

```

Comment:
nit: if you move this line above the previous line and add an empty line on either side, we won't have the way too far to the right formatting for these other lines.

Reply:
Move this line up will change the logic order. But I will add an empty line between this line and the one above to see how that goes. Will update here shortly.

Reply:
Updated by adding some comments between the lines :)

Reply:
Oh, missed that, I just looked and saw it wasn't using EndChar.